releases/v0.14.0.md

```diff
@@ -0,0 +1,89 @@
+# Release Notes for v0.14.0
+
+Release Date: TBD 
+
+This release is based on [3.17.0](https://github.com/kata-containers/kata-containers/releases/tag/3.17.0) of Kata Containers
+and [v0.11.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.11.0) of enclave-cc.
+
+Kata and the CoCo components share an MSRV of 1.80.0.
+
+Please see the [quickstart guide](https://confidentialcontainers.org/docs/getting-started/) or [project documentation](https://confidentialcontainers.org/docs) for more information.
+
+## What's new
+
+* Init-data is supported on bare metal Confidential Containers
+* Image-rs supports registry configuration file for fine-grained proxying and remapping of container registries.
+* KBS Client can be used to set reference values for Trustee.
+* KBS Client has a few simple resource policies built-in.
+* Trustee supports native verification of CCA guests in addition to verification via veraison.
+* Trustee artifacts are built and tested for ARM.
+* Trustee can extract reference values from TCG RIMs.
+* Trustee can be configured to support a larger payload size to accomodate guests with large evidence.
```

Comment:
The confidential guest kernel configuration disabled virtio MMIO transport and rng to reduce host attack surface